emukit/bayesian_optimization/acquisitions/entropy_search.py

```diff
@@ -127,6 +128,9 @@ def update_pmin(self) -> np.ndarray:
         self.logP, self.dlogPdMu, self.dlogPdSigma, self.dlogPdMudMu = epmgp.joint_min(mu, var, with_derivatives=True)
         self.logP = self.logP[:, np.newaxis]
 
+        self.p_min_entropy = np.sum(np.multiply(np.exp(self.logP), np.add(self.logP, self.representer_points_log)),
```

Comment:
Could we maybe add a comment here to make it clear that this is the entropy of the current p_min distribution based on our model

Reply:
Will do

Reply:
Ah sorry my bad you're right. It is a scalar. We only have to average the predicted p_min later over all halluzinated values. It should be correct this way